### PR TITLE
Volunteer test - small refactor

### PR DESCRIPTION
**Pull request by ultrasaurus**
small refactor of https://github.com/18F/openopps-platform/pull/1311

@csarsam what do you think?

### CODE REVIEW

**Comment by csarsam** [comment]
@ultrasaurus Your improvements look good to me - the creatVolunteers utility definitely needed the comment explaining why the creation needs to be serial. (I'd like to know why `findOrCreate` doesn't do what it says on the box when the requests are made at the same time, but it does seem safe to assume that the test timing conditions wouldn't be possible under real usage). I didn't know you could do creation of models using an array of attributes either, that's definitely cleaner.

File: test/integration/models/Volunteer.test.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      }
+    );
+  }
+  return User.create(userAttrs);
+}
+
+function createVolunteers(users, taskId) {
+  // creating 4 volunteers at once, creates 4 badges, not sure why...
+  // but the app never actually does this
+  // var volAttrs = [];
```
Comment (csarsam) [inline test/integration/models/Volunteer.test.js]: This commented out code should get cleaned up.

**Comment by csarsam** [comment]
@ultrasaurus Should this get re-opened against `dev` now that `dev-new` has gotten merged?

**Comment by ultrasaurus** [comment]
yep!  I'll see if I can get this done today for 0.10.1

**Comment by ultrasaurus** [comment]
closing in favor of: https://github.com/18F/openopps-platform/pull/1327

**Pull request closed without merging**